src/bot/ui/printToConsole.js

```diff
@@ -242,15 +243,15 @@ function printToConsole({
 			ui.div(
 				{
 					text: `OUT: ${chalk[simulatedProfit > 0 ? "greenBright" : "red"](
-						toDecimal(route.outAmount, outputToken.decimals)
+						toDecimal(JSBI.toNumber(route.outAmount), outputToken.decimals)
 					)} ${chalk[cache.ui.defaultColor](outputToken.symbol)}`,
 				},
 				{
 					text: " ",
 				},
 				{
 					text: `MIN. OUT: ${chalk.magentaBright(
-						toDecimal(route.outAmountWithSlippage, outputToken.decimals)
+						toDecimal(JSBI.toNumberWithSlippage, outputToken.decimals)
```

Comment:
typo JSBI.toNumberWithSlippage, should be JSBI.toNumber(route.outAmountWithSlippage)